File: c11_svg/B_19_Martin_Angelov.rb
def gime_red_circle x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" stroke="white" stroke-width="2" fill="red" />'
end
def gime_yellow_circle x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" stroke="white" stroke-width="2" fill="yellow" />'
end
def gime_green_circle x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" stroke="white" stroke-width="2" fill="green" />'
end

def rectange x,y,w,h
  '<rect width="'+w.to_s+'" height="'+h.to_s+'" style="fill:rgb(0,0,0);stroke-width:3;stroke:rgb(212,144,200)"/>'
end

File.open("result.svg","w") do |f|
	f.write('<svg xmlns="http://www.w3.org/2000/svg">')
	f.write(rectange 0,0,150,300)
	f.write(gime_red_circle 75,50,40)
	f.write(gime_yellow_circle 75,150,40)
	f.write(gime_green_circle 75,250,40)
	f.write('<text x="30" y="310" font-weight="bold" fill="black">Martin angelov</text>')
	f.write('</svg>')
end

File: c21/B_19_Martin_Angelov.js
var array = [0,1,2,3,4,5,6,7,8,9];
var length = array.length;
var count, count2;

for (count = 0; count < length; count++) { 
    for(count2=0; count2< 5; count2++){
        fill(41, 16, 227);
        rect(count*20,count2*20,20,20);
        fill(10, 10, 10);
        text(""+((count*5)+count2),count*20+(10-textWidth(""+((count*5)+count2)))/2 + 5,count2*20 +15);
    }
}
